src/components/Imip.vue

```diff
@@ -401,7 +401,7 @@ export default {
 				}
 
 				const defaultCalendar = calendarsForPicker.find(
-					(cal) => cal.url === this.currentUserPrincipal.scheduleDefaultCalendarUrl
+					(cal) => cal.url === this.currentUserPrincipal.scheduleDefaultCalendarUrl,
```

Comment:
:eyes:

Reply:
Good catch! Not sure what's going on with eslint. I will just drop the cs commit, it does not even belong here.